exo/api/chatgpt_api.py

```diff
@@ -163,6 +163,8 @@ def __init__(self, node: Node, inference_engine_classname: str, response_timeout
     self.prompts: PrefixDict[str, PromptSession] = PrefixDict()
     self.prev_token_lens: Dict[str, int] = {}
     self.stream_tasks: Dict[str, asyncio.Task] = {}
+    self.default_model = "llama-3.2-1b"
```

Comment:
no need to add this. just change the string `llama-3.1-8b` to `llama-3.2-1b`. Is there any reason we'd want this as a class variable

Reply:
1. One place to change the default_model. Notably, we do the "default model check" twice, once in the function that handles the endpoint, and again in the `parse_chat_request`
2. Also, I thought it could be easier for testing with different default models.